UI/obs-app.cpp

```diff
@@ -1929,13 +1929,19 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 		OBSTranslator translator;
 		program.installTranslator(&translator);
 
-#ifdef _WIN32
 		/* --------------------------------------- */
 		/* check and warn if already running       */
 
 		bool cancel_launch = false;
 		bool already_running = false;
+
+#ifdef _WIN32
 		RunOnceMutex rom = GetRunOnceMutex(already_running);
+#elif defined(__APPLE__)
+		CheckAppWithSameBundleID(already_running);
+#elif defined(__FreeBSD__)
+		// not implemented yet
+#endif
```

Comment:
Is this no-op branch really needed?

Reply:
Hi, @RytoEX, thanks for the comments. Unfortunatly this PR is not fixing whole #3053 and only one part related to macOS, that's why I didn't put "fixes" / "closes" tag. What is the common way in such case?

Reply:
I just wouldn't write an elif no-op branch at the end.  It does nothing, and we can add it later when it's actually implemented.

Reply:
👍 Got it, remove branch and set "fixes" as it seems correct way in this case. Thanks again.